### ISSUE TITLE
Api call signatures

### ISSUE THREAD

**Issue by AlekseiTovkachev** [AlekseiTovkachev/shukersal-deal#58]
Stores:

- [ ] POST api/store/stores
- [ ] GET api/store/stores/{id}
- [ ] PATCH api/store/stores/{id}
- [ ] DELETE api/store/stores/{id}

Products:

- [ ] POST api/store/stores/{storeId}/products
- [ ] GET api/store/stores/{storeId}/products/{productId}
- [ ] GET api/store/products/{productId}
- [ ] PATCH api/store/stores/{storeId}/products/{productId}
- [ ] DELETE api/store/stores/{storeId}/products/{productId}

Categories:

- [ ] GET api/store/categories

Discounts:

- [ ] POST api/store/stores/{storeId}/discountrules
- [ ] DELETE api/store/stores/{storeId}/discountrules

Purchase rules:

- [ ] POST api/store/stores/{storeId}/purchaserules
- [ ] DELETE api/store/stores/{storeId}/purchaserules

Shopping cart:

- [ ] GET /api/shoppingcard/member/{memberId}
- [ ] GET /api/shoppingcart/{cartId}
- [ ] POST /api/shoppingcart/{cartId}/items
- [ ] DELETE /api/shoppingcart/{cartid}
- [ ] PUT /api/shoppingcart/{cartId}

Store manager:

- [ ] POST /api/storemanager/
- [ ] GET /api/storemanager/
- [ ] GET /api/storemanager/stores/{storeId}
- [ ] GET /api/storemanager/member/{memberId}
- [ ] PUT /api/storemanager/{id}
- [ ] DELETE /api/storemanager/{id}

Store manager permissions:

- [ ] POST /api/storemanager/{id}/permissions

Transactions:

- [ ] POST /api/transaction
- [ ] GET /api/transaction/{transactionId}
- [ ] GET /api/transaction/member/{memberId}
- [ ] GET /api/transaction/store/{storeId}
- [ ] DELETE /api/transaction/{transactionId}
- [ ] PUT /api/transaction/{transactionId}

Auth:

- [ ] POST /api/auth/login
- [ ] POST /api/auth/register
- [ ] GET /api/auth/getloggeduser
- [ ] POST /api/auth/changepassword

In short, all calls will lose the word "service" and get a cleanup
After Yuval finishes the discounts and purchase rules their endpoints will be added
